.github/workflows/deploy-faucet.yml

```diff
@@ -0,0 +1,88 @@
+# This action deploys a draft deployment using Netlify to view changes prior to being
+# merged. This does NOT deploy to production site and will only be triggered
+# on changes in the specified paths below.
+name: deploy-faucet 🌐
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches: [develop]
+    paths:
+      - 'apps/faucet/**'
+
+  workflow_dispatch:
+
+env:
+  NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+  NETLIFY_FAUCET_ID: ${{ secrets.NETLIFY_FAUCET_ID }}
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    if: github.ref != 'refs/heads/develop' && github.event.pull_request.draft == false
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+
+      - name: Copy deployment config to root
+        run: cp apps/faucet/netlify.toml netlify.toml
```

Comment:
It looks like we're missing the `apps/faucet/netlify.toml` file. Would you mind helping me add it?